pyproject.toml

```diff
@@ -11,15 +11,15 @@ python = ">=3.8"
 # We set this version explicitly to avoid breaking changes
 opentelemetry-exporter-otlp-proto-http = ">=1.25"
 opentelemetry-sdk = ">=1.25"
-requests = ">=2"
+requests = ">=2.32"
```

Comment:
Can we avoid this? It's going to be installed by customers in their environment and might cause incompatibilities.